src/main/java/org/team1540/robot2025/subsystems/grabber/GrabberConstants.java

```diff
@@ -0,0 +1,12 @@
+package org.team1540.robot2025.subsystems.grabber;
+
+public class GrabberConstants {
+    // TODO: Fix this
+    public static final int GRABBER_ID = 13;
+    public static final int BEFORE_BREAK_CANDI_ID = 1;
+    public static final int AFTER_BREAK_CANDI_ID = 2;
+    public static final String CANDI_BUS = "canivore";
```

Comment:
You don't need this, the CANDi doesn't need to be constructed with the CAN bus name since it should be on the default RIO bus.